### PR TITLE
increase resources

### PR DESCRIPTION
**Pull request by yuvlar**

### CODE REVIEW

**Review by mabuaisha — DISMISSED** [review]

File: jenkins/Jenkinsfile [hunk 1]
```diff
@@ -230,96 +233,99 @@ EOT
...
-                  Write-Host "Installing RabbitMQ..."
-                  # Don't wait for rabbitmq installer as it seems to never return when run silently
-                  Start-Process "${env.RABBITMQ_INSTALLER_PATH}" -ArgumentList /S,/QN -NoNewWindow
-                  for (\$attempt=1; \$attempt -le 60; \$attempt=\$attempt+1) {
-                    \$status=(Get-Service RabbitMQ).Status
-                    if ("\$status" -eq "Running") {
-                      Write-Host "RabbitMQ is running."
-                      Break
+            catchError(message: 'Failure on: cloudify-agent tests', buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+              script{
```
Comment (warolv) [inline jenkins/Jenkinsfile:237]: do we need 'script {' ?

**Review by warolv — CHANGES_REQUESTED** [review]

File: jenkins/Jenkinsfile (hunk 1 above)
Comment (yuvlar) [reply jenkins/Jenkinsfile:237]: I didn't really dig into it, that was working like that before

Comment (warolv) [reply jenkins/Jenkinsfile:237]: ok, so let's remove it, if no need in script{

Comment (yuvlar) [reply jenkins/Jenkinsfile:237]: I'll try to run this pipeline without it and update

**Review by warolv — APPROVED** [review]

**Pull request merged**